lupa/tests/test.py

```diff
@@ -97,7 +98,7 @@ def test_eval(self):
         self.assertEqual(2, self.lua.eval('1+1'))
 
     def test_eval_multi(self):
-        self.assertEqual((1,2,3), self.lua.eval('1,2,3'))
+        self.assertEqual((1, 2, 3), self.lua.eval('1,2,3'))
```

Comment:
All whitespace changes are unrelated to this PR. It's mostly ok to make them (although it makes the git history more difficult to use), but I'd rather have them in a separate PR.

Reply:
> All whitespace changes are unrelated to this PR. It's mostly ok to make them (although it makes the git history more difficult to use), but I'd rather have them in a separate PR.

This is caused by the IDE... Would revert this later.